lib/helpers/find_or_create_students.rb

```diff
@@ -5,16 +5,27 @@ def find_or_create_student(username)
   user_created = nil
   using_cache = !!@user_cache
   if !using_cache || !@user_cache.key?(username)
-    profile = {
-      first_name:             Faker::Name.first_name,
-      last_name:              Faker::Name.last_name,
-      nickname:               username,
-      role_id:                Role.student_id,
-      email:                  "#{username}@doubtfire.com",
-      username:               username,
-      password:               'password',
-      password_confirmation:  'password'
-    }
+    if AuthenticationHelpers.aaf_auth?
+      profile = {
+        first_name:             Faker::Name.first_name,
+        last_name:              Faker::Name.last_name,
+        nickname:               username,
+        role_id:                Role.student_id,
+        email:                  "#{username}@doubtfire.com",
+        username:               username,
+      }
+    else
+      profile = {
+        first_name:             Faker::Name.first_name,
+        last_name:              Faker::Name.last_name,
+        nickname:               username,
+        role_id:                Role.student_id,
+        email:                  "#{username}@doubtfire.com",
+        username:               username,
+        password:               'password',
```

Comment:
Rather than duplicate the hash, why not define it and only add `password` and `password_confirmation` unless `AuthenticationHelpers.aaf_auth?`?

```ruby
profile = { ... }
unless AuthenticationHelpers.aaf_auth?
  profile[:password] = profile[:password_confirmation] = 'password'
end
```